Automattic/wp-calypso
# Light importer flow improvements

#### Proposed Changes

- The UX of the capture screen has been improved. Input field validation, error showing, form submitting, `I don't have a website address` link, etc.
- Improved UX of showing progress from Scanning website to Importing detected colors. (There is a case when scanning last longer than usual, we are aware of it and it will be improved in the future PRs)
- Improved summary screen

#### Testing Instructions

* Go to `/setup/importLight?siteSlug={SLUG}`
* Check if the `Don't have a site address` link redirects you to the importer list and the back button returns back to the capture screen
* Enter a website URL, for example: `https://squealing-cottonmouth.jurassic.ninja/`
* Check if there is progress with showing the palette colors and the final summary screen
* Check if the `Continue` button leads you to the Design picker step

#### Screenshots
<img width="1051" alt="Screenshot 2022-06-30 at 14 15 44" src="https://user-images.githubusercontent.com/1241413/176674912-bdd6cf05-2cf3-4f4a-9614-6b641dd43eda.png">
<img width="1090" alt="Screenshot 2022-06-30 at 14 15 59" src="https://user-images.githubusercontent.com/1241413/176674934-d9d9cca8-8614-4714-89ac-0c6809fd8e88.png">


Related to #64774
